metricbeat/Dockerfile

```diff
@@ -1,21 +1,16 @@
-FROM golang:1.11.2
-MAINTAINER Nicolas Ruflin <ruflin@elastic.co>
+FROM golang:1.10.3
```

Comment:
@sayden Thanks to https://github.com/elastic/beats/pull/11330 we found this change. I wonder if any of the changes here were intended / needed?

Reply:
As far as I remember this we were using `1.10.3` because of needs in the CI (formatting if I remember well) and probably. I don't remember to directly change this, I think it was a co-authored PR with @andrewkroh who maybe can give us some more light here.

Reply:
It might be that things changed during the initial opening of the PR and when it get merged. Could you try a PR to get back the "old" Dockerfile again or have a look on what the actual differences and implications could be?